src/libs/utils/arrangeByLanguage.ts

```diff
@@ -0,0 +1,13 @@
+import * as R from 'ramda';
+import { SupportedLanguage } from '../i18n';
+import { Contributor } from '../domain/contributor';
+import { Notice } from '../domain/notice';
+
+function arrangeByLanguage<Items extends Notice | Contributor>(
```

Comment:
`sortByLanguage` would better describe what it does... or mb `sortByLocale` since we're sorting by locale?